### DISCUSSION TITLE
How do you select colors in Chrome developer tools?

### DISCUSSION THREAD

**Discussion by npearson72** [tailwindlabs/tailwindcss#5569] [Help] [answered]
I have the following style on an element that I inspect in Chrome developer tools:

```
.header_e4q_58yn {
    font-size: 3rem;
    line-height: 1;
    --tw-text-opacity: 1;
    color: rgba(79, 70, 229, var(--tw-text-opacity));
```

Sadly there is no color picker icon that would allow me to adjust the color while developing.

Can anything be done about that?

**Comment by lukewarlow** [comment, marked as answer]
I've filed a [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=1225679) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1718894) bug report for exactly this functionality a few months back.

In the mean time you can manually change the var(--tw-text-opacity) bit to be 1 to play around with the value.

It's worth noting that Safari Web Inspector does show a colour picker when viewing this kind of CSS, so if you're on a Mac you could use Safari web inspector to debug this.

**Reply by npearson72** [reply]
Thanks @lukewarlow will give that a try

**Comment by cheng-dai** [comment]
More than two years later, chrome and Firefox still didn't fix it.🤷
